features/create_sites.feature

```diff
@@ -14,10 +14,15 @@ Feature: Create sites
     And the test_blank/_sass directory should exist
     And the test_blank/assets/css directory should exist
     And the "test_blank/_layouts/default.html" file should exist
-    And the "test_blank/_sass/main.scss" file should exist
+    And the "test_blank/_sass/base.scss" file should exist
```

Comment:
So to be clear, this test was failing with main.scss but doesn't with base.scss? If so, then let's ship this PR!

Reply:
Yes, that's correct.
`dart-sass` and therefore `jekyll-sass-converter-3.0` doesn't allow `css/foo.scss` to import `_sass/foo.scss`.